draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm

```diff
@@ -46,6 +48,18 @@ decoder keyCode =
         Other
 
 
+keyDownDecoder : Decoder Key
+keyDownDecoder =
+    Decode.map keyCodeToKey Html.Events.keyCode
```

Comment:
This is nit picky, could we expose `keyCode` please? :)

Reply:
This was intentional. I wanted to be clear that it's not something from this module.

Reply:
Ah!!!! no probs!